### ISSUE TITLE
JOSM remote from HTTPS no more working with firefox

### ISSUE THREAD

**Issue by frodrigo** [osm-fr/osmose-frontend#111]

**Comment by frodrigo** [comment]
Request https/osmose -> http://localhost

Mixed content:
- FF: fails
- Chromium: fails

Ajax:
- FF: fails
- Chromium: OK

FF: 62.0
Chromium: 69.0

**Comment by frodrigo** [comment]
cc @Marc-marc-marc @don-vip

**Comment by don-vip** [comment]
Looks like https://bugzilla.mozilla.org/show_bug.cgi?id=1488740

**Comment by don-vip** [comment]
I tested old versions of Firefox from 55 (first one to allow mixed content on localhost) up to current one (62). On every version, it does not work for Osmose links, but it works fine from https://www.openstreetmap.org.

I guess Firefox considers the Osmose request as "mixed active content" and blocks it while it considers the OSM request as "Mixed passive/display content" and allows it.

Did it ever work with Osmose before the popup changes? I never tried to access Osmose in HTTPS, I only tested browsers mixed content support from OSM website.
